canonical-websites/www.ubuntu.com
# Sweep /legal and clean up indentation

## Done

- Sweep /legal and clean up indentation
- Remove any redundant classes

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal](http://0.0.0.0:8001/legal)
- Verify everything is as it should
